### PR TITLE
[feature/10.0.0--button] AB#33500 - Closed Radio Gap 

### PR DESCRIPTION
**Pull request by War777**
This PR contains changes to close Radio Input Gaps.
Under [Backlog Item AB#33500](https://dev.azure.com/saddlebackchurch/Church%20Management/_workitems/edit/33500) | Close Inputs gap between Sketch Cloud and cm-ui library
- [TASK AB#37328](https://dev.azure.com/saddlebackchurch/Church%20Management/_workitems/edit/37328/) | Radio - Gaps
- [TASK AB#37329](https://dev.azure.com/saddlebackchurch/Church%20Management/_workitems/edit/37329/) | Radio Pill Group - Gaps
- [TASK AB#37340](https://dev.azure.com/saddlebackchurch/Church%20Management/_workitems/edit/37340/) | Radio - Docs
- [TASK AB#37341](https://dev.azure.com/saddlebackchurch/Church%20Management/_workitems/edit/37341/) | Radio Pill Group - Docs
- [TASK AB#37346](https://dev.azure.com/saddlebackchurch/Church%20Management/_workitems/edit/37346/) | Radio - Unit Tests
- [TASK AB#37347](https://dev.azure.com/saddlebackchurch/Church%20Management/_workitems/edit/37347/) | Radio Pill Group - Unit Tests

### CODE REVIEW

**Comment by morethanfire** [comment]
Do not merge, this PR is based on `feature/10.0.0--button` right now. We need to wait until that branch is merged and then change this base branch to another.

File: docs/src/app/navigationConstants.js
Comment (morethanfire) [inline docs/src/app/navigationConstants.js:259]: Please sort these two `levelFour` objects alphabetically.

File: docs/src/inputs/radio/api/api.jsx
Comment (morethanfire) [inline docs/src/inputs/radio/api/api.jsx:16]: Please remove `propTypes`. It is not being used.

File: docs/src/inputs/radio/radio.jsx
Comment (morethanfire) [inline docs/src/inputs/radio/radio.jsx:37]: Change "Radio" to "Standard Radio Buttons"

File: docs/src/inputs/radio/radio.jsx
Comment (morethanfire) [inline docs/src/inputs/radio/radio.jsx:41]: Change "A simple radio" to "General use"

File: docs/src/inputs/radio/examples/radioSample.jsx
Comment (morethanfire) [inline docs/src/inputs/radio/examples/radioSample.jsx:26]: Please put in a Grid. Example can be followed here: `./docs/src/inputs/button/examples/exampleButtonSolid.jsx`

File: docs/src/inputs/radio/examples/radioSamplePill.jsx
Comment (morethanfire) [inline docs/src/inputs/radio/examples/radioSamplePill.jsx:27]: Please put in a Grid. Example can be followed here: `./docs/src/inputs/button/examples/exampleButtonSolid.jsx`

File: src/inputs/radio/radio.jsx
Comment (morethanfire) [inline src/inputs/radio/radio.jsx:36]: Change comment to: "Override or extend the styles applied to Input."

File: src/inputs/radio/radio.jsx
Comment (morethanfire) [inline src/inputs/radio/radio.jsx:50]: Change comment to: "Assign additional class names to Radio."

File: src/inputs/radio/radio.jsx
Comment (morethanfire) [inline src/inputs/radio/radio.jsx:180]: Please change to `typography.pxToRem(16)`

File: src/inputs/radio/radio.jsx
Comment (morethanfire) [inline src/inputs/radio/radio.jsx:513]: Please use `UI_CLASS_NAME` from `./src/global/constants.js` instead of 'ui'.

**Review by morethanfire — CHANGES_REQUESTED** [review]
Thanks @War777, this is looking good. Please have a look through the requested changes.

File: docs/src/app/navigationConstants.js
Comment (War777) [reply docs/src/app/navigationConstants.js:259]: done!

File: docs/src/inputs/radio/api/api.jsx
Comment (War777) [reply docs/src/inputs/radio/api/api.jsx:16]: Done!

File: docs/src/inputs/radio/radio.jsx
Comment (War777) [reply docs/src/inputs/radio/radio.jsx:37]: done!

File: docs/src/inputs/radio/radio.jsx
Comment (War777) [reply docs/src/inputs/radio/radio.jsx:41]: done!

**Comment by cbrewer1689** [comment]
The do not merge label was removed because PR #232 was merged allowing this PRs base to be changed from `feature/10.0.0--button` to `feature/cumulative--10.0.0`.

File: docs/src/inputs/radio/examples/radioSample.jsx
Comment (War777) [reply docs/src/inputs/radio/examples/radioSample.jsx:26]: done!

File: docs/src/inputs/radio/examples/radioSamplePill.jsx
Comment (War777) [reply docs/src/inputs/radio/examples/radioSamplePill.jsx:27]: done!

File: src/inputs/radio/radio.jsx
Comment (War777) [reply src/inputs/radio/radio.jsx:36]: done!

File: src/inputs/radio/radio.jsx
Comment (War777) [reply src/inputs/radio/radio.jsx:50]: done!

File: src/inputs/radio/radio.jsx
Comment (War777) [reply src/inputs/radio/radio.jsx:180]: done!

File: src/inputs/radio/radio.jsx
Comment (War777) [reply src/inputs/radio/radio.jsx:513]: done!

**Comment by JLeeC** [comment]
PR Review Checkpoint Meeting Thurs 2020-11-10

@cbrewer1689  to re-review. Target: 11/12/2020

**Review by cbrewer1689 — APPROVED** [review]
Nice! Thank you @War777.

:shipit:

**Pull request merged**
